app/controllers/openid_token_proxy/callback_controller.rb

```diff
@@ -0,0 +1,20 @@
+module OpenIDTokenProxy
+  class CallbackController < ApplicationController
+    def handle
+      unless code = params[:code]
+        render nothing: true, status: :bad_request
```

Comment:
We should perhaps consider rendering some text to describe why it's a bad request (like "Required parameter 'code' missing").